DDDEastAnglia/Views/Account/_ExternalLoginsPartial.cshtml

```diff
@@ -0,0 +1,49 @@
+﻿@model IEnumerable<DDDEastAnglia.Models.LoginMethodViewModel>
+
+@if (Model.Any())
+{
+    foreach (var loginMethod in Model)
+    {
+        <div style="margin-bottom: 10px;">
```

Comment:
Any suggestions for a name? I couldn't come up with anything that was meaningful, which is part of the reason why I left them inline.